velox/docs/functions/spark/misc.rst

```diff
@@ -15,6 +15,11 @@ Miscellaneous Functions
         SELECT at_least_n_non_nulls(2, 0, 1.0, NULL);  -- true
         SELECT at_least_n_non_nulls(2, 0, array(NULL), NULL);  -- true
 
+.. spark:function:: get_struct_field(input, ordinal) -> T
+
+    Returns the value of field in the row type ``input``.
+    The field position is specified by ``ordinal``.
+
```

Comment:
It would be nice to provide some examples, including the basic one, nested complex type and invalid ordinal.

Reply:
Maybe you miss some comments @WangGuangxin

Reply:
> Maybe you miss some comments @WangGuangxin

@jinchengchenghh done